cvpysdk/sedstype.py

```diff
@@ -0,0 +1,35 @@
+
+class SEDSType(object):
```

Comment:
Will this be used across multiple files, or in a single file.
If it'll used in a single file, then please add this dictionary to that file only.

Also, for this operation, I don't think a class is required.
You can directly create the dict here, and take a copy of it in your class